PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEmcalJetHdEdxCorrelations.h

```diff
@@ -244,6 +251,7 @@ namespace PWGJE
       Double_t fMinSharedMomentumFraction;    ///< Minimum shared momentum with matched jet
       bool fRequireMatchedPartLevelJet;       ///< True if matched jets are required to be matched to a particle level jet
       Double_t fMaxMatchedJetDistance;        ///< Maximum distance between two matched jets
+      TFile *fEPcorrectionFile;               //!<! File containing the EP corrections
```

Comment:
This needs to be streamed (`///<`) - otherwise, it won't work on the grid. Probably simpler would be to read the tree and stream that instead, but as you like.

In either case, you'll need to bump your claasdef

Reply:
Thanks, I have made the modifications.